sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnInvoker.java

```diff
@@ -136,7 +136,7 @@
     InputT element(DoFn<InputT, OutputT> doFn);
 
     /** Provide a link to the input element. */
```

Comment:
Can you update the docstring here to explain what index is? It might also be useful to update the `element` docstring as well - "Provide a _link_ to the input element." seems misleading, from what I can tell the implementations should just return a reference to the element.

Reply:
Done